components/login/login_controller/login_controller.jsx

```diff
@@ -62,14 +62,20 @@ class LoginController extends React.Component {
             login: PropTypes.func.isRequired,
             addUserToTeamFromInvite: PropTypes.func.isRequired,
         }).isRequired,
-    }
+    };
 
     constructor(props) {
         super(props);
 
         let loginId = '';
-        if ((new URLSearchParams(this.props.location.search)).get('extra') === Constants.SIGNIN_VERIFIED && (new URLSearchParams(this.props.location.search)).get('email')) {
-            loginId = (new URLSearchParams(this.props.location.search)).get('email');
+        if (
```

Comment:
You are reformating a lot of code to and adding a lot of line breaks which is inconsistent with the current code styling. I would prefer to keep untouch the code that is not related to the PR in that sense, actually the code should be consistent with the rest fo the code base, and this "80 characters" approach is breaking the current existing style.

Reply:
Sorry, I've misconfigured my VSCode eslint settings that auto-formats the code and ignores the `.eslintrc`. I'll fix this first thing tomorrow.